node/src/tests/test-PlainTransport.ts

```diff
@@ -316,6 +316,18 @@ test('router.createPlainTransport() with non bindable IP rejects with Error', as
 		.toThrow(Error);
 }, 2000);
 
+test('router.createPlainTransport() with enableMulticast succeeds', async () =>
+{
+	// Use default cryptoSuite: 'AES_CM_128_HMAC_SHA1_80'.
+	const transport1 = await router.createPlainTransport(
+		{
+			listenIp   : '127.0.0.1',
+			enableMulticast: true
+		});
+
+	expect(typeof transport1.id).toBe('string');
```

Comment:
Do not do this useless assertion here. Instead use `expect(await xxxxxx).not.toThrow();`.

Also, you are creating a `transport1` here but you are not closing it once the test ends, so it remains alive until all tests end. Please close the transport1 here.